examples/blockchain/src/network.rs

```diff
@@ -233,130 +95,200 @@ impl aleph_bft::Network<NetworkData> for Network {
 }
 
 pub struct NetworkManager {
-    swarm: Swarm<Behaviour>,
+    id: usize,
+    address: Address,
+    addresses: Vec<Option<Address>>,
+    bootnodes: HashMap<u32, Address>,
```

Comment:
You could keep addresses in a `HashMap` and bootnodes in `HashSet`. I guess this would make stuff simpler. You would not have to create a vec of `None`.

Reply:
fixec